hazelcast/include/hazelcast/client/internal/socket/BaseSocket.h

```diff
@@ -336,7 +338,12 @@ class BaseSocket : public hazelcast::client::socket
     boost::asio::ip::tcp::resolver& resolver_;
     T socket_;
     int32_t call_id_counter_{ 0 };
-    typedef std::deque<std::vector<boost::asio::const_buffer>> Outbox;
+    struct outbox_entry_t{
```

Comment:
`_t` suffix appended is generally appended if it is alias of a type. So I would recommend to delete it. For sake of simplicity, `entry` is pretty understandable.

```suggestion
    struct entry
```

Reply:
fixed